CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v7.0 (in development) ###
+
+* Replaces pg_dist_shard_placement metadata table with pg_dist_placement
```

Comment:
That does sound reasonable. Could you check with @jasonmp85 on the right way to do this?

(iirc, the build scripts parse the changelog and are fairly strict about versioning and such)